### PR TITLE
HDDS-7127. Fix Transparent Data Encryption documentation

### PR DESCRIPTION
**Pull request by myskov**
## What changes were proposed in this pull request?

The following docs issues are fixed:
* hadoop.security.key.provider.path must be set up via core-site.xml, not hdfs-site.xml
* encKey" is invalid name for hadoop kms key. Uppercase characters are not supported for keys

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7127

## How was this patch tested?

Unit tests are passed. No additional testing needed for documentation change.

### CODE REVIEW

File: hadoop-hdds/docs/content/security/SecuringTDE.zh.md [hunk 1]
```diff
@@ -42,14 +42,14 @@ hadoop.security.key.provider.path  | KMS uri. <br> 比如 kms://http@kms-host:96
...
   上面这个命令会创建一个用于保护桶数据的密钥。创建完成之后，你可以告诉 Ozone 在读写某个桶中的数据时使用这个密钥。
 
    * 将加密密钥分配给桶
 
   ```bash
-  ozone sh bucket create -k encKey /vol/encryptedbucket
+  ozone sh bucket create -k enckey /vol/encryptedbucket
   ```
 
 这条命令执行后，所以写往 _encryptedbucket_ 的数据都会用 encKey 进行加密，当读取里面的数据时，客户端通过 KMS 获取密钥进行解密。换句话说，Ozone 中存储的数据一直是加密的，但用户和客户端对此完全无感知。
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/security/SecuringTDE.zh.md:55]:
```suggestion
这条命令执行后，所以写往 _encryptedbucket_ 的数据都会用 enckey 进行加密，当读取里面的数据时，客户端通过 KMS 获取密钥进行解密。换句话说，Ozone 中存储的数据一直是加密的，但用户和客户端对此完全无感知。
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @myskov for the fix.  LGTM, except one leftover reference to `encKey`.

**Comment by myskov** [comment]
Thanks @adoroszlai, I applied you suggestion commit

**Review by adoroszlai — APPROVED** [review]
Thanks @myskov for updating the patch.

**Pull request merged**